.env.example

```
@@ -475,3 +475,4 @@ LINEAR_API_KEY=
 LINEAR_TEAM_ID=
 # Game Feedback label ID in Linear (optional - find via Linear API or create a new label)
 LINEAR_GAME_FEEDBACK_LABEL_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LINEAR_GAME_FEEDBACK_LABEL_ID key should go before the LINEAR_TEAM_ID key

(UnorderedKey)